inc/composer/class-installer.php

```diff
@@ -6,8 +6,18 @@
 use Composer\Package\PackageInterface;
 
 class Installer extends BaseInstaller {
-	public function getInstallPath( PackageInterface $package, $framework_type = '' ) {
 
+	/**
+	 * Check if the installer supports a given type.
+	 *
+	 * @param string $type
+	 * @return bool
+	 */
+	public function supports( $type ) {
+		return in_array( $type, [ 'wordpress-plugin', 'wordpress-muplugin' ], true );
```

Comment:
Do we need `wordpress-dropin` in this list too?

Reply:
We don't currently specify it in https://github.com/humanmade/altis-core/blob/master/inc/composer/class-installer.php#L42 so we haven't needed it up to this point.